boost-common/src/main/java/boost/common/config/BoostProperties.java

```diff
@@ -19,24 +19,27 @@
 
 public final class BoostProperties {
 
+    // Boost specific
+    public static final String BOOST_PROP_PREFIX = "boost_";
+
     // HTTP Endpoint properties
-    public static final String ENDPOINT_HOST = "boost.http.host";
-    public static final String ENDPOINT_HTTP_PORT = "boost.http.port";
-    public static final String ENDPOINT_HTTPS_PORT = "boost.http.securePort";
+    public static final String ENDPOINT_HOST = "boost_http_host";
+    public static final String ENDPOINT_HTTP_PORT = "boost_http_port";
+    public static final String ENDPOINT_HTTPS_PORT = "boost_http_securePort";
 
     // Datasource default properties
-    public static final String DATASOURCE_PREFIX = "boost.db.";
-    public static final String DATASOURCE_DATABASE_NAME = "boost.db.databaseName";
-    public static final String DATASOURCE_SERVER_NAME = "boost.db.serverName";
-    public static final String DATASOURCE_PORT_NUMBER = "boost.db.portNumber";
-    public static final String DATASOURCE_USER = "boost.db.user";
-    public static final String DATASOURCE_PASSWORD = "boost.db.password";
-    public static final String DATASOURCE_CREATE_DATABASE = "boost.db.createDatabase";
-    public static final String DATASOURCE_URL = "boost.db.url";
+    public static final String DATASOURCE_PREFIX = "boost_db_";
+    public static final String DATASOURCE_DATABASE_NAME = "boost_db_databaseName";
+    public static final String DATASOURCE_SERVER_NAME = "boost_db_serverName";
+    public static final String DATASOURCE_PORT_NUMBER = "boost_db_portNumber";
+    public static final String DATASOURCE_USER = "boost_db_user";
+    public static final String DATASOURCE_PASSWORD = "boost_db_password";
+    public static final String DATASOURCE_CREATE_DATABASE = "boost_db_createDatabase";
+    public static final String DATASOURCE_URL = "boost_db_url";
 
-    public static final String AES_ENCRYPTION_KEY = "boost.aes.key";
+    public static final String AES_ENCRYPTION_KEY = "boost_aes_key";
 
-    public static final String INTERNAL_COMPILER_TARGET = "boost.internal.compiler.target";
+    public static final String INTERNAL_COMPILER_TARGET = "boost_internal_compiler_target";
```

Comment:
It might be slightly better to leave this one alone.. since it's not an external,not part of the app+server config.

Reply:
ok, reverting that one back to dots